Docs/packer seo functions batch3 (#13284)
hashicorp/packer: website/content/docs/templates/hcl_templates/functions/index.mdx

---
page_title: Built-in functions reference overview
description: |-
  Functions built into HCL language can transform and combine values. Learn about using built-in functions in your Packer templates. 
---

# Built-in functions overview

> This topic provides an overview of the built-in HashiCorp configuration language (HCL) functions you can use to create Packer templates. 

Reviewer: ```suggestion
This topic provides an overview of the built-in HashiCorp configuration language (HCL) functions you can use in your Packer templates. 
```

## Functions syntax

> You can use functions in Packer templates to transform and combine values. The general

Reviewer: ```suggestion
You can use functions in Packer templates to transform and combine values. The 
```
